### PR TITLE
amap: add page

### PR DESCRIPTION
**Pull request by ciph3rz**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/amap.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# amap
+
+> Next-generation scanning tool for pentesters.
+> More information: <https://www.thc.org/>.
```
Comment (navarroaxel) [inline pages/linux/amap.md:4]:
```suggestion
> More information: <https://www.thc.org>.
```

File: pages/linux/amap.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Scan port 80 on target IP address and print ASCII banner response, omit closed ports, verbosely:
```
Comment (navarroaxel) [inline pages/linux/amap.md:6]: But this command doesn't have a `-v` argument
```suggestion
- Scan port 80 on a target host and print ASCII banner response, omit closed ports, with detailed information:
```

File: pages/linux/amap.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`amap -bvq {{ip_or_hostname}} {{80}}`
+
+- Scan port 1-1024 and capture banners:
+
+`amap -bvq {{ip_or_hostname}} {{1-1024}} -B`
```
Comment (navarroaxel) [inline pages/linux/amap.md:12]: We prefer long arguments if available instead of `-B`

Comment (ciph3rz) [reply pages/linux/amap.md:12]: Unfortunately application does not support long arguments

**Comment by ciph3rz** [comment]
Reference to #6863

File: pages/linux/amap.md (hunk 2 above)
Comment (navarroaxel) [reply pages/linux/amap.md:6]: Which one fits better?
```suggestion
- Scan port 80 on a target host and print ASCII banner response, omit closed ports:
```

File: pages/linux/amap.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# amap
+
+> Next-generation scanning tool for pentesters.
+> More information: <https://www.thc.org>.
+
+- Scan port 80 on a target host and print ASCII banner response, omit closed ports:
```
Comment (navarroaxel) [inline pages/linux/amap.md:6]:
```suggestion
- Scan the port 80 on a target host and print ASCII banner response, omitting closed ports:
```

File: pages/linux/amap.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`amap -bqv {{ip_or_hostname}} {{80}}`
+
+- Scan port 1-1024 and capture banners:
```
Comment (navarroaxel) [inline pages/linux/amap.md:10]:
```suggestion
- Scan the ports 1-1024 and capture banners:
```

File: pages/linux/amap.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+`amap -bqv {{ip_or_hostname}} {{1-1024}} -B`
+
+- Scan port 1-1024 and analyze the banners results efficiently:
```
Comment (navarroaxel) [inline pages/linux/amap.md:14]: How is an `analyze` non-`efficiently`? :thinking:

File: pages/linux/amap.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+`amap -bqv {{ip_or_hostname}} {{1-1024}} -A`
+
+- Scan port 1-1024 without banner grabbing and return open ports:
```
Comment (navarroaxel) [inline pages/linux/amap.md:18]:
```suggestion
- Scan the ports 1-1024 without banner grabbing and display open ports:
```

File: pages/linux/amap.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
+# amap
+
+> Next-generation scanning tool for pentesters.
+> More information: <https://www.thc.org>.
+
+- Scan port 80 on a target host and print ASCII banner response, omit closed ports:
+
+`amap -bqv {{ip_or_hostname}} {{80}}`
```
Comment (EmilySeville7cfg) [inline pages/linux/amap.md:8]:
```suggestion
- Scan port 80 on a target host and print ASCII banner response, omit closed ports:

`amap -bq --verbose {{ip|hostname}} {{80}}`
```

Warning: [mnemonics can't be suggested as parsing `amap` man page failed: no `-b`, `-q` options found, possible broken man page](https://manned.org/amap.1).
Mandatory fix: [long option must be used if it's portable](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax).
Mandatory fix: [`or` keyword is undefined for anything expect `file_or_directory`, wrong syntax used](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths).

File: pages/linux/amap.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+
+- Scan port 1-1024 and capture banners:
+
+`amap -bqv {{ip_or_hostname}} {{1-1024}} -B`
```
Comment (EmilySeville7cfg) [inline pages/linux/amap.md:12]:
```suggestion
- Scan a specific port and capture banners:

`amap -bqv {{ip|hostname}} {{1..1024}} -B`
```

Almost the same issue as above.
Warning: mnemonics can't be suggested for options as parsing man page failed and possibly several short options differing just with case detected.
Optional fix: `specific` may sound better.
Optional fix: `..` can be used for ranges, our Emacs extension accepts just such syntax as correct (standardization).

File: pages/linux/amap.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
+- Scan port 1-1024 and analyze the banners results efficiently:
+
+`amap -bqv {{ip_or_hostname}} {{1-1024}} -A`
```
Comment (EmilySeville7cfg) [inline pages/linux/amap.md:16]:
```suggestion
- Scan a specific port and analyze the banners results efficiently:

`amap -bqv {{ip|hostname}} {{1..1024}} -A`
```
The same issue as above.

File: pages/linux/amap.md [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+
+- Scan port 1-1024 without banner grabbing and return open ports:
+
+`amap {{ip_or_hostname}} {{1-1024}} -P`
```
Comment (EmilySeville7cfg) [inline pages/linux/amap.md:20]:
```suggestion
- Scan a specific port without banner grabbing and return open ports:

`amap {{ip|hostname}} {{1..1024}} -P`
```
The same issue as above.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
Mandatory fixes found, changes requested.
Thanks for a contribution! 🚀

**Pull request closed without merging**